trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js

```diff
@@ -144,14 +149,33 @@ const submitDealAfterUkefIds = async (dealId, dealType, checker) => {
 
 exports.submitDealAfterUkefIds = submitDealAfterUkefIds;
 
+const dealHasAllUkefId = async (dealId) => {
```

Comment:
small niggle, can we add an S please `dealHasAllUkefIds`

Reply:
Can we also move this to the helpers directory, and add unit tests?

Reply:
Have added.